SRC/Makefile.am

```diff
@@ -28,7 +28,7 @@ libarpack@LIBSUFFIX@_la_SOURCES = $(SSRC) $(DSRC) $(CSRC) $(ZSRC)
 libarpack@LIBSUFFIX@_la_SOURCES += $(top_builddir)/dbgini.f
 libarpack@LIBSUFFIX@_la_SOURCES += $(top_builddir)/staini.f
 libarpack@LIBSUFFIX@_la_LIBADD = $(top_builddir)/UTIL/libarpackutil.la $(LAPACK_LIBS) $(BLAS_LIBS)
-libarpack@LIBSUFFIX@_la_LDFLAGS = -no-undefined -version-info 2:0
+libarpack@LIBSUFFIX@_la_LDFLAGS = -no-undefined -version-info 3:0:1
```

Comment:
With the 3 in the version? I am not sure it will

Reply:
Libtool has its own versioning scheme (current:revision:age, not major.minor.patch).  Note that the digits are separated by colons rather than dots.  3:0:1 gets converted to 2.1.0.
https://www.gnu.org/software/libtool/manual/html_node/Updating-version-info.html

Reply:
The syntax is awkward but I think libtool designed it that way on purpose, to prevent you from filling it in with $PACKAGE_VERSION and force you to think about compatibility.  Please try building it yourself to double-check that it creates libarpack.so.2.1.0